### ISSUE TITLE
Template add-on for paid content

### ISSUE THREAD

**Issue by Ivshti** [Ivshti/stremio-addons#25]
This is important for content creators, figure out how to make a template add-on for paid content.

**Comment by Ivshti** [comment]
A possible solution is to use the ability to put content in iframes, and create an example server for a paywalled content with a paypal form

**Comment by jaruba** [comment]
maybe also load all iframes passing a get variable with the unique user id for one time payments?

**Comment by jaruba** [comment]
i have nothing against an iframe solution, facebook started their facebook apps with iframes too

**Comment by Ivshti** [comment]
Yeah, seems reasonable. The thing is, if you want paid content, you will design your own pay-wall, with your own protection. So it makes sense. Providing a template also makes sense if you can't design your own
